### DISCUSSION TITLE
exporting metadata

### DISCUSSION THREAD

**Discussion by kactaplb** [Difegue/LANraragi#960] [Q&A] [answered]
Are there any plans to have a plugin that exports metadata to either `comicinfo.xml`, `series.json` or some other format? All the different formats are all a mess, but it's better than your data being locked into a single ecosystem. 

If not, I might do it myself.

**Comment by Difegue** [comment, marked as answer]
Not currently, since after all, how would you even decide which format is the best one to export to?  
People would argue `comicinfo.xml` is the closest there is to a standard, but writing XML is a pain.  

As far as I'm concerned, the current backup JSON exports are easy to re-ingest into another system, so I don't think there's any lock-in in place here.

**Reply by kactaplb** [reply]
Ah I somehow missed the backup feature. Yea the json is plenty good enough thanks.
